packages/eslint-plugin/src/configs/strict-type-checked.ts

```diff
@@ -76,5 +76,6 @@ export = {
     '@typescript-eslint/triple-slash-reference': 'error',
     '@typescript-eslint/unbound-method': 'error',
     '@typescript-eslint/unified-signatures': 'error',
+    '@typescript-eslint/use-unknown-in-catch-callback-variable': 'error',
```

Comment:
just double checking/confirming we want to add this to the strict config before the next major?
i don't think we've ever expressly discussed our strategy for the strict configs.
they're not technically included in our [versioning policy](https://typescript-eslint.io/users/versioning#eslint-plugin) for a major release if they're changed - but I wonder if adding a brand new rule in a minor might be a bit disruptive for users?

cc @JoshuaKGoldberg

Reply:
Yup confirmed, we've done this before (#7957, #8011). Since the strict config is explicitly out of semver, users are opting into this.